content/02.introduction.md

```diff
@@ -44,82 +57,70 @@ The current pandemic of COVID-19, caused by the severe acute respiratory syndrom
 The possibility of asymptomatic cases is also being investigated.
 The transmission and mortality rate estimations of COVID-19 remain to be determined.
 
-[this section may be better as a figure panel + legend]
-Phylogenetic analysis of a PCR amplicon fragment from five patients along with the total virus genome of 29.8 kilobases indicates that the virus is a novel betacoronavirus belonging to the B lineage, also known as sarbecovirus.
-The sarbecovirus lineage also includes the human SARS coronavirus [@doi:10/ggjs7j].
-
+#### Transmission and Evolution
 
-#### Virion, morphology, structure:
-Coronavirus virions are spherical with diameters ranging between 100 to 160nm.
-The virion has a lipid envelope with spike (S) glycoproteins on the surface creating a distinctive "crown" shape, for which the family of viruses was named [@isbn:0781760607].
-
-
-#### Genome structure and replication:
-Coronavirus genomes are single-stranded non-segmented positive RNA (ssRNA+) that are 27 to 32kb in length.
-The SARS-CoV-2 genome lies in the middle of this range at 29,903 bp in length [@doi:10/ggjr43].
-Coronavirus genomes are comprised of a replicase gene that is 2/3 of the genome and the remaining 1/3 that encodes structural proteins including: spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins.
-There are additional accessory genes that are present depending on the strain.
-The non-structural polypeptide is translated into 16 non-structural proteins (nsp1-16, except Gammacoronaviruses that do not contain nsp1), which form replication machinery to synthesize viral RNA [@doi:10.1002/jmv.25681].
+SARS-CoV-2 is a novel human coronavirus, following SARS-CoV and MERS-CoV to become the third coronavirus causing severe pneumonia in humans to emerge in the 21st century [@doi:10.1016/bs.aivir.2018.01.001]. 
+The evolutionary origins of the SARS-CoV-2 virus are not yet fully understood.
```

Comment:
Earlier it says that SARS-CoV2 is from the sarbecovirus lineage...I am assuming evolutionary lineage to mean phylogenetic lineage. Do you mean something else here?

https://www.biorxiv.org/content/10.1101/2020.03.30.015008v1.abstract

Reply:
I didn't write this text but I'm inclined to agree with you! I think they meant the origins re: intermediate hosts. What about:
```suggestion
The proximal origins of the SARS-CoV-2 virus are not yet fully understood, but genomic analyses have placed it within the coronavirus phylogeny.
```

Reply:
This might just be me, but I'm not sure what "proximal origin" means in this context, so the only edit I would make is to just have `The intermediate host of the SARS-CoV-2...` But I default to you as you have been reading WAY more on this subject

Reply:
I think it refers to the most recent branch in which you can map an organism too. For example the most distal origin of humans would be the last common universal ancestor and the most proximal would be homo.

Reply:
```suggestion
Though the intermediate host serving as the source for the zoonotic introduction of SARS-CoV-2 to human populations has not yet been identified, the SARS-CoV-2 virus has been placed within the coronavirus phylogeny through comparative genomic analyses.
```
@ajlee21 I was going for what @nilswellhausen described, but it would be better if this is broadly understandable! I think this sentence is a bit clunky, but does it do a better job of getting the idea across (for now?)